src/main/java/climatecontrol/ClimateChooser.java

```diff
@@ -1,8 +1,8 @@
 
-package climateControl;
```

Comment:
Please do not rename the packages (not even to all lower-case), it will break any compat other mods may have with this.